src/app/index.js

```diff
@@ -353,7 +353,9 @@ const reloadMenu = () => ipcRenderer.send('pls-reload-menu')
 window.addEventListener('load', reloadMenu)
 window.addEventListener('focus', reloadMenu)
 
-ipcRenderer.on('new-project', () => createBlankProj())
+ipcRenderer.on('new-project', () => {
+  store.dispatch(actions.project.startCreatingNewProject(true))
```

Comment:
@jeanawhou, I think you missed this one.  Wouldn't this set the template to `true`?

Reply:
@Quiescent thank you for checking 🙇‍♀️